code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -373,6 +378,60 @@ Code:
 						menu += " [S[A.equipment+1]] [S[A.lighting+1]] [S[A.environ+1]] [add_lspace(A.lastused_total, 6)]  [A.cell ? "[add_lspace(round(A.cell.percent()), 3)]% [chg[A.charging+1]]" : "  N/C"]<BR>"
 
 				menu += "</FONT></PRE>"
+		if (53)
+			menu = "<h4><img src=pda_alert.png> Please select an Alert Computer</h4><BR>"
+			alertmonitor = null
+			alertmonitors = list()
+			var/alertcount = 0
+
+
+			for(var/obj/machinery/computer/station_alert/aMon in world)

```

Comment:
because I don't know how to isolate the search to a specific Z-level other than by checking the Z-level after finding it.